### PR TITLE
draft add insurer name in csv export

### PR DESCRIPTION
**Pull request by konavivekramakrishna**
## Proposed Changes

- add insurer name in patients csv export
### Associated Issue
- https://github.com/coronasafe/care_fe/issues/6517

image

@coronasafe/care-backend-maintainers @coronasafe/care-backend-admins

### CODE REVIEW

File: care/facility/models/patient.py [hunk 1]
```diff
@@ -530,6 +530,9 @@ def annotate_diagnosis_ids(*args, **kwargs):
         "last_consultation__new_discharge_reason": "Reason for discharge",
         "last_consultation__discharge_date": "Date of discharge",
         "last_consultation__discharge_date__time": "Time of discharge",
+        
+        # insurance details
+         "policy__insurer_name": "Insurer Name",
```
Comment (rithviknishad) [inline care/facility/models/patient.py:535]: What happens if there are multiple insurance policies?

Comment (konavivekramakrishna) [reply care/facility/models/patient.py:541]: @rithviknishad 

when trying to import policy, getting error as circular dependency how to solve this.

Comment (rithviknishad) [reply care/facility/models/patient.py:541]: In the function passed as the value in CSV_MAKE_PRETTY for this, you can import it the model inside the function.

@sainak is there a better solution?

Comment (sainak) [reply care/facility/models/patient.py:541]: we can move this to patient viewset

**Pull request closed without merging**